upi/openstack/network.yaml

```diff
@@ -11,23 +11,23 @@
   gather_facts: no
 
   tasks:
-  - name: 'Create the cluster network'
+  - name: 'Create the primary cluster network'
     os_network:
       name: "{{ os_network }}"
 
-  - name: 'Set the cluster network tag'
+  - name: 'Set tags on the  primary cluster network'
     command:
-      cmd: "openstack network set --tag {{ cluster_id_tag }} {{ os_network }}"
+      cmd: "openstack network set --tag {{ primary_cluster_network_tag }} --tag {{ cluster_id_tag }} {{ cluster_network.network.id }}"
```

Comment:
What about cluster_network, where is it defined?
Why changing from {{ os_network }} to {{ cluster_network.network.id }}?

Reply:
my bad. fixing.